apps/site/components/Common/AvatarGroup/Avatar/index.module.css

```diff
@@ -15,9 +18,21 @@
     dark:text-neutral-300;
 }
 
-.avatarRoot {
+.container {
```

Comment:
-1 for using `container` because it's already a tailwind class

Reply:
I totally forgot about that, thank you 🎉